web/src/main/resources/assets/img/wheelchair.svg

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
There is something in the middle of the wheel. Not visible with white background:

![image](https://user-images.githubusercontent.com/129644/64976132-e20f2d80-d8b0-11e9-863a-d0b00b34df06.png)

![image](https://user-images.githubusercontent.com/129644/64976181-f3f0d080-d8b0-11e9-92f8-f65a4c0fc61d.png)


Sorry, if this is a bit nickpicking but can you remove/change this?